app/code/Magento/SalesGraphQl/etc/schema.graphqls

```diff
@@ -0,0 +1,19 @@
+# Copyright © Magento, Inc. All rights reserved.
+# See COPYING.txt for license details.
+
+type Query {
+    customerOrders: Orders @resolver(class: "Magento\\SalesGraphQl\\Model\\Resolver\\Orders") @doc(description: "List of customer orders")
+}
+
+type Order @doc(description: "Order mapping fields") {
+    id: Int
+    increment_id: String
+    created_at: String
+    grant_total: Float
+    state: String
```

Comment:
Why do we need to expose state on the storefront?

Reply:
I will tend to provide a lot of info on the order, so the query can be made to extract just you need.

Reply:
State is an internal concept, on storefront we normally expose only Status. 
```suggestion
```

Reply:
Thanks for info. I changed already